src/widgets/menu/defaultTemplates.tsx

```diff
@@ -7,17 +7,40 @@ import { cx } from '../../lib/utils';
 import type { MenuComponentTemplates } from './menu';
 
 const defaultTemplates: MenuComponentTemplates = {
-  item({ cssClasses, url, label, count }) {
+  item({
+    cssClasses,
+    url,
+    count,
+    highlighted,
+    label,
+    isFromSearch,
+    searchable,
+  }) {
     return (
       <a className={cx(cssClasses.link)} href={url}>
-        <span className={cx(cssClasses.label)}>{label}</span>
+        {searchable ? (
+          <span
+            className={cx(cssClasses.label)}
+            dangerouslySetInnerHTML={
+              isFromSearch ? { __html: highlighted } : undefined
+            }
```

Comment:
there should be a todo here about using the highlight component, like in refinementList